packages/tenable_io/data_stream/scanner/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,190 @@
+---
+description: Pipeline for processing Tenable.io scanner logs.
+processors:
+  - set:
+      field: ecs.version
+      value: '8.7.0'
+  - set:
+      field: event.category
+      value: [host]
+  - set:
+      field: event.type
+      value: [info]
+  - rename:
+      field: message
+      target_field: event.original
+      ignore_missing: true
+  - json:
+      field: event.original
+      target_field: json
+  - set:
+      field: event.kind
+      value: state
+  - set:
+      field: event.ingested
+      value: '{{_ingest.timestamp}}'
+  - set:
+      field: host.hostname
+      copy_from: json.hostname
+      ignore_empty_value: true
+  - set:
+      field: host.name
+      copy_from: json.name
+      ignore_empty_value: true
+  - set:
+      field: host.ip
+      copy_from: json.ip_addresses
+      ignore_empty_value: true
+  - remove:
+      field: host.ip
+      if: ctx.host?.ip != null && ctx.host?.ip[0] == ""
```

```suggestion
      if: ctx.host?.ip != null && ctx.host.ip[0] == ""
```
(similar below; when we have reached past the conjunction, we know `ctx.host` must be non-null)